### PR TITLE
Ci/docker

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
## Description
The docker files used hard-wired addresses for common environments. Both, frontend and backend, can now use environment variables instead:
- ci: Dockerfile for frontend uses substring mechanism to update environment variables in built files
- ci: Dockerfile for frontend uses nginx unpriviledged
- ci: Dockerfile for backend now can use environment variables for application.properties
- fix: application.properties were outdated.
- ci: added docker-compose for both
- docs: updated INSTALL.mds

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: frontend/start-nginx.sh
Comment (mhellmeier) [inline frontend/start-nginx.sh:1]: A license header is missing.

File: frontend/replace_env.sh
Comment (mhellmeier) [inline frontend/replace_env.sh:1]: Why should this empty file be committed in the `main` branch?

**Review by mhellmeier — CHANGES_REQUESTED** [review]

**Review by mhellmeier — CHANGES_REQUESTED** [review]
Remember to update the `DEPENDENCIES_BACKEND`. Based on the pipeline, the file isn't up to date.

File: frontend/replace_env.sh
Comment (tom-rm-meyer-ISST) [reply frontend/replace_env.sh:1]: removed

**Review by mhellmeier — APPROVED** [review]
LGTM, since the failed quality check pipeline is addressed in https://github.com/eclipse-tractusx/sig-infra/issues/230

**Pull request merged**